### PR TITLE
Registry stuff

### PR DESCRIPTION
**Pull request by shadowfacts**
Remapped most of the registry stuff.

Used Mojang names where possible (only `DefaultMappedRegistry`)

### CODE REVIEW

File: mappings/net/minecraft/util/registry/IDRegistry.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{
+  "obf": "db",
+  "name": "net/minecraft/util/registry/IDRegistry",
```
Comment (asiekierka) [inline mappings/net/minecraft/util/registry/IDRegistry.json]: IdRegistry. We don't capitalize both letters of ID.

**Comment by asiekierka** [comment]
Could you squash the two commits? I normally wouldn't ask that, but the change here is so small a squashing would be good for future viewers.

**Comment by shadowfacts** [comment]
GitHub can [squash and merge](https://github.com/blog/2141-squash-your-commits).

**Comment by asiekierka** [comment]
Woah.

**Pull request merged**